src/systems/si-typographic/include/units/isq/si/typographic/length.h

```diff
@@ -38,11 +38,16 @@ namespace units::isq::si::typographic {
 // TODO Conflicts with (https://en.wikipedia.org/wiki/Pica_(typography)), verify correctness of below conversion factors
 // and provide hyperlinks to definitions
 struct pica_comp :
-    named_scaled_unit<pica_comp, "pica(comp)", as_magnitude<ratio(4233333, 1000000, -3)>(), si::metre> {};
-struct pica_prn : named_scaled_unit<pica_prn, "pica(prn)", as_magnitude<ratio(2108759, 500000, -3)>(), si::metre> {};
+    named_scaled_unit<pica_comp, "pica(comp)", as_magnitude<4'233'333>() * pow<-9>(as_magnitude<10>()), si::metre> {};
```

Comment:
I am not sure, but maybe `4'233'333 * as_magnitude<10, -9>()` could work out of the box (`ratio` is implicitly constructible from integral type. If not another overload for plain numbers could be added to `magnitude`.

Reply:
This is very desirable, but I think @JohelEGP's comment in the other thread suggests it won't work.

Reply:
To make `magnitude` an NTTP type similar to `ratio` we would need a compile-time vector, right? Also, it would probably make compile-time errors even harder to analyze?

Reply:
> To make `magnitude` an NTTP type similar to `ratio` we would need a compile-time vector, right?

Yes. I'm working on something like that.

> Also, it would probably make compile-time errors even harder to analyze?

In depends. I think it has the potential to make them clearer.